pkg/kubelet/kubelet.go

```diff
@@ -2617,3 +2617,12 @@ func isSyncPodWorthy(event *pleg.PodLifecycleEvent) bool {
 	// ContainerRemoved doesn't affect pod state
 	return event.Type != pleg.ContainerRemoved
 }
+
+// PrepareResources calls container Manger PrepareResources API
+// This method implements RuntimeHelper interface
+func (kl *Kubelet) PrepareResources(pod *v1.Pod, container *v1.Container) error {
```

Comment:
Likewise here. Once again though -- this currently only wraps the `containerManager.PrepareResources()` call and could be expanded in the future so maybe it's OK. In any case, I don't have a better suggestion for a name off the top of my head.